{
  "Header": {
    "Original": "CVE-2015-20107 (In Python (aka CPython) up to 3.10.8, the mailcap module does not add  ...)",
    "Line": 86947,
    "ID": "CVE-2015-20107",
    "Description": "(In Python (aka CPython) up to 3.10.8, the mailcap module does not add  ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3432-1}",
      "Line": 86948,
      "Type": "xref",
      "Bugs": [
        "DLA-3432-1"
      ]
    },
    {
      "Original": "- python3.10 3.10.6-1",
      "Line": 86949,
      "Type": "package",
      "Package": "python3.10",
      "Kind": "fixed",
      "Version": "3.10.6-1"
    },
    {
      "Original": "- python3.9 \u003cremoved\u003e",
      "Line": 86950,
      "Type": "package",
      "Package": "python3.9",
      "Kind": "removed"
    },
    {
      "Original": "[bullseye] - python3.9 \u003cno-dsa\u003e (Minor issue)",
      "Line": 86951,
      "Type": "package",
      "Release": "bullseye",
      "Package": "python3.9",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "- python3.7 \u003cremoved\u003e",
      "Line": 86952,
      "Type": "package",
      "Package": "python3.7",
      "Kind": "removed"
    },
    {
      "Original": "[buster] - python3.7 \u003cno-dsa\u003e (Minor issue)",
      "Line": 86953,
      "Type": "package",
      "Release": "buster",
      "Package": "python3.7",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "- python3.5 \u003cremoved\u003e",
      "Line": 86954,
      "Type": "package",
      "Package": "python3.5",
      "Kind": "removed"
    },
    {
      "Original": "[stretch] - python3.5 \u003cno-dsa\u003e (Minor issue)",
      "Line": 86955,
      "Type": "package",
      "Release": "stretch",
      "Package": "python3.5",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "- python2.7 \u003cunfixed\u003e",
      "Line": 86956,
      "Type": "package",
      "Package": "python2.7",
      "Kind": "unfixed"
    },
    {
      "Original": "[bullseye] - python2.7 \u003cignored\u003e (Python 2.7 in Bullseye not covered by security support)",
      "Line": 86957,
      "Type": "package",
      "Release": "bullseye",
      "Package": "python2.7",
      "Kind": "ignored",
      "Description": "Python 2.7 in Bullseye not covered by security support"
    },
    {
      "Original": "[stretch] - python2.7 \u003cno-dsa\u003e (Minor issue)",
      "Line": 86958,
      "Type": "package",
      "Release": "stretch",
      "Package": "python2.7",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://bugs.python.org/issue24778",
      "Line": 86959,
      "Type": "NOTE",
      "Description": "https://bugs.python.org/issue24778"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/issues/68966",
      "Line": 86960,
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/issues/68966"
    },
    {
      "Original": "NOTE: https://github.com/python/cpython/pull/91993",
      "Line": 86961,
      "Type": "NOTE",
      "Description": "https://github.com/python/cpython/pull/91993"
    }
  ]
}